source/content/tls-compatibility.md

```diff
@@ -0,0 +1,127 @@
+
+<Alert title="What's affected?" type="info">
+
+**Application requests** include things like cURL or remote HTTP requests (e.g. `wp_remote_get` in WordPress) against a server or service. They do _not_ include requests for remotely hosted JavaScript or CSS files that are loaded when a webpage is rendered. (Your web browser handles the TLS handshake for these types of requests.)
+
+</Alert>
+
+### How do I check TLS compatibility for my site?
+
+You can check your whether site's codebase makes HTTP connections to servers that support TLS 1.2 or 1.3 by using the WordPress TLS Compatibility Checker plugin or via the command line.
+
+<TabList>
+
+<Tab title="WordPress" id="check-compatibility-wp" active={true}>
+
+WordPress users can use the [Pantheon WP TLS Compatibility Checker](https://github.com/pantheon-systems/wp-tls-compatibility-checker) to scan their site for outgoing TLS connections. The plugin can be run from the command line using WP-CLI or in the admin via a new TLS Compatibility page under the Tools menu. If any URLs are found that do not support TLS 1.2 or 1.3 connections, a list is provided on the TLS Compatibility page (or shown in the terminal).
+
+The plugin also adds issues to the Site Health screen after the tests have been run if URLs were found that do not support TLS 1.2+ connections.
+
+View the project [README](https://github.com/pantheon-systems/wp-tls-compatibility-checker/blob/main/README.md) for more detailed information about customizing the checks, installation and usage.
+
+</Tab>
+
+<Tab title="Drupal" id="check-compatibility-drupal">
+
+Drupal users can use the [Pantheon Drupal TLS Compatibility Checker](https://github.com/pantheon-systems/drupal_tls_checker) to scan their site for outgoing TLS connections. This module supports Drupal 10/11 and can be run from the command line using Drush or in the admin at `/admin/config/development/tls-checker`. If any URLs are found in the `modules` or `themes` directories that do not support TLS 1.2 or 1.3 connections, a list is provided on that page (or shown in the terminal).
+
+View the project [README](https://github.com/pantheon-systems/drupal_tls_checker/blob/main/README.md) for more detailed information about customizing the checks, installation and usage.
+
+</Tab>
+
+<Tab title="Command Line" id="check-compatibility-cli">
+
+Individual URLs can be checked for TLS 1.2/1.3 compatibility with the `curl` command.
+
+```bash{promptUser: user}
+curl --tlsv1.2 --head --connect-timeout 10 "<your url to check>"
+```
+
+If you receive a 200 status code in the response (e.g. `HTTP/2 200`) along with header information, the check succeeded. If you receive an error (e.g. `TLS connect error`), it means the server you are attempting to make a connection to does not support that version of TLS.
+
+A more robust example that scans all the files in a particular path can be found [in this Gist](https://gist.github.com/scottbuscemi/8915d6d53f34f5012406ef5d21b222a1).
```

Comment:
@scottbuscemi okay to link to your gist?

Reply:
I don't think that's a sustainable choice for the docs. Better to just copy it into the docs.